registration/models.py

```diff
@@ -60,6 +60,12 @@ def __str__(self) -> str:
         """Represent OKUser by e-mail address."""
         return self.email
 
+    class Meta:
+        """Define the message IDs."""
+
+        verbose_name = _('User')
+        verbose_name_plural = _('Users')
```

Comment:
As far as I remember, the addition of new names would also lead to a new migration generated. Could you please check for that and add it if necessary?